bpx/parsers.py

```diff
@@ -1,20 +1,25 @@
 from bpx import BPX
 
 
-def parse_bpx_file(filename: str) -> BPX:
+def parse_bpx_file(filename: str, v_tol: float = 0.001) -> BPX:
```

Comment:
we should also add `v_tol` to `parse_bpx_obj` and `parse_bpx_str`

Reply:
This is a bit tricky because if I overwrite `parse_obj` in `ExtraBaseModel` class, I'll break other methods that call `parse_obj`.

What would be the most natural and 'future-proof' way of introducing parameter `v_tol` here?

Reply:
We could use have some global settings, e.g. have a file

```python3
class Settings(object):
    tolerances = {
        "Voltage [V]": 1e-3,  
    }

settings = Settings()
```

then do `bpx.settings.tolerances["Voltage [V]"] = v_tol` to update the tolerance before reading a BPX? Maybe there is a cleaner solution.